packages/mineral-ui-tokens/tokens/generic/backgroundColor.json

```diff
@@ -39,6 +39,10 @@
     { "name": "backgroundColor_warningPrimary",        "value": "{!bronze_60}" },
     { "name": "backgroundColor_warningPrimary_active", "value": "{!bronze_70}" },
     { "name": "backgroundColor_warningPrimary_focus",  "value": "{!bronze_60}" },
-    { "name": "backgroundColor_warningPrimary_hover",  "value": "{!bronze_50}" }
+    { "name": "backgroundColor_warningPrimary_hover",  "value": "{!bronze_50}" },
+    { "name": "backgroundColor_grayscalePrimary",          "value": "{!gray_20}" },
+    { "name": "backgroundColor_grayscalePrimary_active",   "value": "{!gray_20}" },
+    { "name": "backgroundColor_grayscalePrimary_focus",    "value": "{!gray_20}" },
+    { "name": "backgroundColor_grayscalePrimary_hover",    "value": "{!gray_30}" }
```

Comment:
still missing:
grayscale_active
grayscale_focus
grayscale_hover

We want the hollow/secondary grayscale buttons to have the same color scheme.

Reply:
also, we're not really giving it a scale. you're using 20, 20, 20, 30
it should be more like: 20, 30, 20, 10 to match what mineral-ui is doing with the other variants.

Reply:
Agree. Will fix color ramp to reflect